main.go

```diff
@@ -179,6 +182,17 @@ func main() {
 	renewDeadline := time.Second * time.Duration(leaderLeaseRenewDeadline)
 	retryPeriod := time.Second * time.Duration(leaderLeaseRetryPeriod)
 
+	// filter cache to subscribe to events from specific configmaps
+	newCache := cache.BuilderWithOptions(cache.Options{
+		SelectorsByObject: cache.SelectorsByObject{
+			&corev1.ConfigMap{}: {Field: fields.Set{
+				"metadata.name":      config.VpcCniConfigMapName,
+				"metadata.namespace": config.KubeSystemNamespace,
```

Comment:
1. why don't using cache.Options.Namespace?
2. let's use separate variable for LeaderElectionNamespace and maybe `VpcCNIConfigMapNamespace` instead of `KubeSystemNamespace` since technically they might be different in the future.

Reply:
cache.Options.Namespace will restrict the cache for watching all other resources. 
SelectorsByObject will restrict it per GVK as we only want to filter cache for the ConfigMap. 

I've changed the namespace variables.